### PR TITLE
Add Changes to Contract ReadMe and Main ReadMe

### PR DESCRIPTION
**Pull request by jebitok-dev**
In this I:
- Added Changes to the Contract ReadMe and Main ReadMe file (Any recommendation or changes can be added or share)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+- Solidity
+- Foundry
+- Typescript
+- Nextjs
+- OpenZeppelin's ERC721
+## Getting Started 
 
 Steps:
 
 1. Generate Merkle tree of recipients by following README in [generator/](https://github.com/Anish-Agnihotri/merkle-airdrop-starter/tree/master/generator)
```
Comment (sorXCode) [inline README.md:18]:
```suggestion
1. Generate Merkle tree of recipients by following README in [generator/](https://github.com/sorxcode/merkle-airdrop-starter-ERC721/tree/master/generator)
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
-2. Setup and deploy MerkleClaimERC20 contracts by following README in [contracts/](https://github.com/Anish-Agnihotri/merkle-airdrop-starter/tree/master/contracts)
+2. Setup and deploy MerkleClaimERC721 contracts by following README in [contracts/](https://github.com/Anish-Agnihotri/merkle-airdrop-starter/tree/master/contracts)
```
Comment (sorXCode) [inline README.md:19]:
```suggestion
2. Setup and deploy the MerkleClaimERC721 contract by following README in [contracts/](https://github.com/sorxcode/merkle-airdrop-starter-ERC721/tree/master/contracts)
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,49 @@
...
 3. Setup and deploy front-end by following README in [frontend/](https://github.com/Anish-Agnihotri/merkle-airdrop-starter/tree/master/frontend)
```
Comment (sorXCode) [inline README.md:20]:
```suggestion
3. Setup and deploy front-end by following README in [frontend/](https://github.com/sorxcode/merkle-airdrop-starter-ERC721/tree/master/frontend)
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,49 @@
...
 
+### Contributors
+- [Cohort VI - Web3Bridge](Web3bridge.com)
+### How to Contribute
+- Fork the repository
+- Git clone https://github.com/your-username/merkle-airdrop-starter-ERC721
+- git checkout -b branch name
+- git remote add upstream https://github.com/jebitok-dev/merkle-airdrop-starter-ERC721
```
Comment (sorXCode) [inline README.md:28]: should we keep the upstream to the current repo?

```suggestion
- git remote add upstream https://github.com/sorxcode/merkle-airdrop-starter-ERC721
```

File: README.md [hunk 5]
```diff
@@ -1,13 +1,49 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgment
+- We acknowledge the creators of the [Merkle Airdrop Starter](https://github.com/sorXCode/merkle-airdrop-starter-ERC721) and [Web3Bridge](https://www.web3bridge.com/) for project instructions.
```
Comment (sorXCode) [inline README.md:44]: The original creator of the Merkle Airdrop Starter is @Anish-Agnihotri , should we link this to the original repo?

```suggestion
- We acknowledge the creators of the [Merkle Airdrop Starter](https://github.com/Anish-Agnihotri/merkle-airdrop-starter) and [Web3Bridge](https://www.web3bridge.com/) for project instructions.
```

File: contracts/README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
-# MerkleClaimERC20
+# Web3Bridge MerkleClaimERC721 Contract
 
-ERC20 token claimable by members of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree). Useful for conducting Airdrops. Utilizes [Solmate ERC20](https://github.com/Rari-Capital/solmate/blob/main/src/tokens/ERC20.sol) for modern ERC20 token implementation.
+MerkleClaimERC721 contract that utilizes [OpenZeppelin's ERC721URIStorage, ERC721, MerkleProof and Ownable](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721) for modern ERC721 token implementation. It defines ```Claim, hasClaimed, UpdateMerkleRoot ```. It is claimable by members of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree). Useful for conducting Web3Bridge Certificate Airdrops.
```
Comment (sorXCode) [inline contracts/README.md:3]:
```suggestion
MerkleClaimERC721 contract that utilizes [OpenZeppelin's ERC721URIStorage, ERC721, MerkleProof and Ownable](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721) for modern ERC721 token implementation. It defines `claim`, `hasClaimed`, `updateMerkleRoot`. It is claimable by members of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree). Useful for conducting Web3Bridge Certificate Airdrops. 
```

File: contracts/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
-# MerkleClaimERC20
+# Web3Bridge MerkleClaimERC721 Contract
 
-ERC20 token claimable by members of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree). Useful for conducting Airdrops. Utilizes [Solmate ERC20](https://github.com/Rari-Capital/solmate/blob/main/src/tokens/ERC20.sol) for modern ERC20 token implementation.
+MerkleClaimERC721 contract that utilizes [OpenZeppelin's ERC721URIStorage, ERC721, MerkleProof and Ownable](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721) for modern ERC721 token implementation. It defines ```Claim, hasClaimed, UpdateMerkleRoot ```. It is claimable by members of a [Merkle tree](https://en.wikipedia.org/wiki/Merkle_tree). Useful for conducting Web3Bridge Certificate Airdrops. 
 
 ## Test
```
Comment (sorXCode) [inline contracts/README.md:6]:
```suggestion
To run the test, run

forge test
```

**Review by sorXCode — COMMENTED** [review]
Nice work @jebitok-dev. Kindly check the comments/suggestions, thanks.

**Comment by jebitok-dev** [comment]
Thank you for your suggestions

**Pull request merged**